BrainPortal/app/models/en_cbrain_local_data_provider.rb

```diff
@@ -104,6 +104,11 @@ def impl_provider_rename(userfile,newname)  #:nodoc:
     return false unless FileUtils.move(oldpath.to_s,newpath.to_s)
     true
   end
+  
+  # this returns the category of the data provider -- used in view for admins
+  def self.pretty_category_name
+    "Enhanced CBRAIN Types"
```

Comment:
I think its clearer to return it with the type. It also makes for less work to add it later in the controller.